Makefile

```diff
@@ -4,27 +4,31 @@ SHELL=/bin/bash -euo pipefail -O globstar
 
 all:
 	make clean > build.log
+	make check-licenses >> build.log
 	make build >> build.log
 	make test >> build.log
 	make release >> build.log
 
-.PHONY: install build test publish release clean
+.PHONY: all
 
 install: install-validator install-node install-python install-hooks
 
 build: build-specification build-coordinator build-validator build-proxies
 
-test: validate-models check-licenses test-coordinator
+test: validate-models test-coordinator
```

Comment:
removed to remove duplicate check-licenses from pipeline

Reply:
Probably missing something but I can't see where else check-licenses is being called from

Reply:
Good spot. Think I misread what was going on here, I thought the utils build called it, but it doesn't. Added this back.